ipc/provider/src/lib.rs

```diff
@@ -41,6 +37,15 @@ pub mod manager;
 const DEFAULT_REPO_PATH: &str = ".ipc";
 const DEFAULT_CONFIG_NAME: &str = "config.toml";
 
+/// The vm type the dry run performs in
+#[derive(Debug, Clone, Copy, strum::EnumString)]
+pub enum VMType {
+    #[strum(serialize = "Evm", serialize = "evm")]
```

Comment:
`clap` has its own `ValueEnum` type. Why do we need `strum` here? 

What does it do with multiple `serialize` options?

Reply:
Ah, different crate :confused: